dev-tools/mage/settings.go

```diff
@@ -41,7 +41,7 @@ const (
 	fpmVersion = "1.13.1"
 
 	// Docker images. See https://github.com/elastic/golang-crossbuild.
-	beatsFPMImage = "docker.elastic.co/beats-dev/fpm"
+	beatsFPMImage = "docker.elastic.co/observability-ci/fpm"
```

Comment:
this change will be reverted once https://github.com/elastic/golang-crossbuild/pull/531 has been merged

Reply:
now reverted